src/themes/ec/variables/_radio.scss

```diff
@@ -14,19 +14,17 @@ $radio: (
   help-text-color: var(--c-d),
   help-text-font: var(--f-s),
   invalid-border-color: var(--c-er),
-  invalid-border-color-hover: var(--c-er),
+  invalid-border-color-hover: color-mix(
+      in srgb,
+      var(--cm-surface-dark) 50%,
+      var(--c-er)
+    ),
   label-color: var(--c-d),
-  hover-circle: (
-    height: 2.5rem,
-    width: 2.5rem,
-    content: '',
-    display: block,
-    border-radius: 50%,
-    position: absolute,
-    top: 50%,
-    left: 50%,
-    transform: translate(-50%, -50%),
-    background-color: var(--c-p-20),
-    z-index: -1,
+  hover-effect: (
```

Comment:
we can probably simplify this a little; here we are using a single nested element, and only for one thing
This may be a simple parameter like 'box-color-hover'. If we define it in EC and EU, we won't have to test for it in the scss